### DISCUSSION TITLE
Can't run queries in template-view?

### DISCUSSION THREAD

**Discussion by Sininho** [stdword/logseq13-full-house-plugin#35] [Q&A] [answered]
Hello. I wanted to run a query as a template-view. The test template  is just:  
```
{{query [[TestPage]] }}
template:: test
```
![image](https://github.com/stdword/logseq13-full-house-plugin/assets/29395551/e7284be0-2157-44a1-a850-535d26e588da)

But when I call : `{{renderer :template-view, test}}`,  instead giving me the query results, it just shows the block text. It even shows the block's hidden properties!![image](https://github.com/stdword/logseq13-full-house-plugin/assets/29395551/bc438ea8-f96e-4e29-82d1-17605842feab)
Am I doing something wrong, or is it the normal behavior? 

The reason why I wanted the view instead of template is that I repeat it in a lot of pages and I sometimes change the query, and I want it to update accordingly. 

I am using the desktop version of logseq on linux (in case it matters).

Thank you for any help, and thanks to the developer for developing the plugin!

**Comment by stdword** [comment, marked as answer]
Hello! Thank you for the kind words 🙏

Queries are not supported in the plugin: this is not a bug.
Implementing this feature means complete reimplementing of query table: there is no special API to reuse Logseq existed functionality.
There is an issue to collect votes for these feature: #20 
But, to be honest, it is a big piece of work, and the chances of it being implemented are small 😢

What you can do for you case is to use [embedded queries](https://discuss.logseq.com/t/make-advanced-queries-reusable/15281/5?u=stdword).
Or special case for [parametrised query](https://discuss.logseq.com/t/passing-parameter-values-into-a-query/17170/3?u=stdword).

**Reply by Sininho** [reply]
Thank you for the quick and helpful answer! Thank you again for the plugin, and I wish you happy holidays :-)
